test/integration/integration_test.go

```diff
@@ -48,6 +53,10 @@ var (
 
 	testDir string
 
+	//proxy
+	proxyListener net.Listener
+	proxyLog      *bytes.Buffer
```

Comment:
agree, but maybe after we know what goes wrong in CI. the benefit of having it in `integration_test.go` is the possible use of `println` ;-)